### ISSUE TITLE
Couple of minor points about combat information

### ISSUE THREAD

**Issue by Sanglorian** [Tuxemon/Tuxemon#1967]
**Is your feature request related to a problem? Please describe.**
Just a few minor issues relating to how combat information is displayed: 

* The message about XP gained should come after the message about the monster being knocked out. 
* When a monster avoids capture, is it possible to not go on to the next stage of the battle until the player has clicked Enter/Shift? At the moment, the capture fails and the monster that's the target of the capture immediately attacks, making it a little hard to follow what's happening. 
* It's fantastic that the Tuxepedia page about the captured monster pops up after the capture occurs - it really adds something. Is it possible to make Enter/Shift make the page disappear, not just escape? 

Thanks for considering these!

**Comment by Grimmys** [comment]
> It's fantastic that the Tuxepedia page about the captured monster pops up after the capture occurs - it really adds something. Is it possible to make Enter/Shift make the page disappear, not just escape?

Looks like pressing "Shift" is already making the page disappear.
Also, these controls look the same in any other place a Tuxepedia page is opening.
Meaning that adding "Enter" as an alternative key for closing the page after the capture, would also likely add it for the case where the page is shown directly from the Tuxepedia entry menu, or for example when the player is checking the Tuxemons that are in the bins.

I have no issue with this though, I think it could be fine to have the "Enter" key to close the page in every cases.
